code/game/objects/items/stacks/medical.dm

```diff
@@ -105,12 +111,13 @@
 /obj/item/stack/medical/bruise_pack
 	name = "bruise pack"
 	singular_name = "bruise pack"
-	desc = "A therapeutic gel pack and bandages designed to treat blunt-force trauma."
+	desc = "A therapeutic gel pack and bandages designed to treat minor blunt-force trauma."
 	icon_state = "brutepack"
 	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
-	heal_brute = 40
-	self_delay = 20
+	heal_brute = 30
+	brute_heal_penalty = 0.4
+	self_delay = 15

```

Comment:
just set /obj/item/stack/medical to 15 so both bruise_pack and ointment can inherit.

It makes no sense to set it as a value you never use anyways :P